File: app/src/main/java/org/nervos/neuron/item/NeuronDApp/TakePhotoItem.java
package org.nervos.neuron.item.NeuronDApp;

/**
 * Created by BaojunCZ on 2018/11/5.
 */
public class TakePhotoItem extends BaseNeuronDAppCallbackItem {

    public Info info;

    public TakePhotoItem(String status, String errorCode, String errorMsg, String imagePhoto) {
        this.status = status;
        this.errorCode = errorCode;
        this.errorMsg = errorMsg;
        info = new Info();
        info.imagePhoto = imagePhoto;
    }

    public class Info {
        public String imagePhoto;
    }
}

File: app/src/main/java/org/nervos/neuron/item/NeuronDApp/TakePhotoItem.kt
package org.nervos.neuron.item.NeuronDApp

/**
 * Created by BaojunCZ on 2018/11/5.
 */
class TakePhotoItem(status: String, errorCode: String, errorMsg: String, imagePhoto: String) :
        BaseNeuronDAppCallbackItem(status, errorCode, errorMsg) {

    var info: Info

    init {
        info = Info()
        info.imagePhoto = imagePhoto
    }

    inner class Info {
        var imagePhoto: String? = null
    }
}
